src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/DocumentControllerTest.java

```diff
@@ -26,7 +26,6 @@
 import uk.gov.hmcts.reform.civil.model.docmosis.sealedclaim.Representative;
```

Comment:
Shouldnt there be a test at the Controller Layer to test these changes...

Reply:
you're right!! I refactored the documentControllerTest class as you suggest